<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>141</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MiToAi</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="materialButton">
       <property name="toolTip">
        <string>Select mental ray materials to replace with arnold</string>
       </property>
       <property name="text">
        <string>Material Selection</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="meshButton">
       <property name="statusTip">
        <string>Select meshes to replace their mental ray with arnold</string>
       </property>
       <property name="text">
        <string>Mesh Selection</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="selectionLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Select mesh(es) from the scene</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="ignoreButton">
       <property name="toolTip">
        <string>If checked, the gamma correct node will not be used in the resulting network</string>
       </property>
       <property name="text">
        <string>Ignore Gamma Correct</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="removeGammaButton">
       <property name="toolTip">
        <string>If checked, the gamma nodes will be removed from the scene</string>
       </property>
       <property name="text">
        <string>Remove Gamma Correct</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="removeMiButton">
     <property name="toolTip">
      <string>If checked, the Mi material nodes will be removed from the scene</string>
     </property>
     <property name="text">
      <string>Remove Mi Nodes</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="createButton">
       <property name="text">
        <string>Create Arnolds</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
